src/Psalm/Internal/PhpVisitor/Reflector/ClassLikeNodeScanner.php

```diff
@@ -386,14 +389,14 @@ public function start(PhpParser\Node\Stmt\ClassLike $node): ?bool
             }
         }
 
-        foreach ($node->getComments() as $comment) {
-            if (!$comment instanceof PhpParser\Comment\Doc) {
+        foreach ($node->getComments() as $doc_comment) {
+            if (!$doc_comment instanceof PhpParser\Comment\Doc) {
```

Comment:
I didn't actually mean to change `$comment` to `$doc_comment` and `$comments` to `$parsed_docblock` across the entire class, but after I did I figured it'd be better if it was consistent, so I kept it.